From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@seanlarge__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

seanlarge commented:
also, anywhere it says 'see npm-install' the link is wrong, this is the correct link: https://docs.npmjs.com/cli/install

tysonn commented:
@rmcmurray - Could you review this PR in the public repo from @seanlarge?

cephalin commented:
@seanlarge Thanks for improving the Azure documentation!
@tysonn @ggailey777 Please go ahead and merge.

carolinacmoravia committed:
M20356: Unnecessary word: "either"

Hello, @Drewm3,

Translator has reported possible source content issue. The word "either" seems to be added incorrectly.   

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

mimig1 committed:
Cosmos DB: Removed media folder from unused documentdb folder

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Update gallery listing doc

chetansriv committed:
Update app gallery listing

chetansriv committed:
Updated app gallery listing doc

Commit:
added info about support and apt-transport-https and removed WSL from note to inline.

v-nagta committed:
USER STORY 1219373 SaaS App Tutorial: Slack Tutorial Update

Commit:
automation : monitor tutorial updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into vmtutorialmonitorlinux

Commit:
first update for linux monitoring tutorial

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
corrected spelling

Commit:
updates

Commit:
removing old images

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

v-nagta committed:
updated

v-viinde committed:
USER STORY 1220283 SaaS App Tutorial: ADP Update

v-viinde committed:
Merge branch 'master' into adpv1

Commit:
fixes for images

Commit:
forgot border

Commit:
automation: update runbook execution powershell

Commit:
updates

Commit:
automation : correct headings

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Update app gallery tutorial

Commit:
Update TOC for unified speech service; add overview doc

barbkess committed:
update metadata

barbkess committed:
update topic type

barbkess committed:
metadata updates

v-nagta committed:
update

v-viinde committed:
Updated

v-geberr committed:
update - new ux ddl

v-geberr committed:
removed fuzzy search

roygara committed:
Reassigning articles to correct owner.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Reassign

mimig1 committed:
Cosmos DB: Updated screenshots DocDB->Cosmos DB

roygara committed:
Fixing assignment.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Reassign

roygara committed:
Grabbing linux variants.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Reassign

roygara committed:
Capturing missing docs.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Reassign

v-geberr committed:
remove pngx file

v-shils committed:
Merge pull request #36594 from v-geberr/0329-qs-prebuilt-fuzzy-search

removed fuzzy search

cherylmc committed:
ubiquiti

PRMerger17 committed:
Merge pull request #31324 from carolinacmoravia/patch-546

M20356: Unnecessary word: "either"

iainfoulds committed:
[VM] Updating for simplified PowerShell cmdlets

PRMerger18 committed:
Merge pull request #36568 from barbkess/metadata

Metadata

Commit:
Reassign articles and update manager metadata

v-shils committed:
Merge pull request #36591 from v-geberr/0329-qs-simple-entity

update - new ux ddl

MicrosoftGuyJFlo committed:
[MFA] GitHub Issue 6460 Modern auth and Office 2013

rjagiewich committed:
Merge pull request #36595 from roygara/Reassign

Taking ownership of managed disk content

SnehaGunda committed:
Removing editor

cherylmc committed:
date

cherylmc committed:
format

PRMerger20 committed:
Merge pull request #36519 from georgewallace/autobugs

automation : correct headings

iainfoulds committed:
[VM] Updating metadata

KumudD committed:
fixed figure labels

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into lboverview0329

SnehaGunda committed:
Updating author field

SnehaGunda committed:
Updating manager metadata

GitHubber17 committed:
Merge pull request #36402 from georgewallace/vmtutorialmonitorlinux

automation : update vm monitoring tutorial for linux

SnehaGunda committed:
Updating token details

GitHubber17 committed:
Merge pull request #36517 from georgewallace/jobdocupdate

automation : update job execution doc

mimig1 committed:
additional screenshot fix

PRMerger10 committed:
Merge pull request #36606 from KumudD/lboverview0329

Updated image descriptions

ktoliver committed:
Merge pull request #36604 from kraigb/kraigb-vs-azure-tools-reassign

Reassign articles and update manager metadata

v-geberr committed:
keys concept cleanup

Commit:
Revert changes for unified speech project

v-geberr committed:
update concept text

Commit:
Update author field in existing articles

ggailey777 committed:
Added the experiment ID into ms.custom

Commit:
fix unicode garbage in rank-results.md

curtand committed:
[Azure AD LICENSING] add preview boilerplate

Commit:
Merge pull request #36355 from v-nagta/slack

USER STORY 1219373 SaaS App Tutorial: Slack Tutorial Update

Commit:
Merge pull request #36461 from v-viinde/adpv1

USER STORY 1220283 SaaS App Tutorial: ADP Update

Commit:
Merge pull request #36613 from ggailey777/experiment

Added the experiment ID into ms.custom

anirudhcavale committed:
Grafana national cloud update

anirudhcavale committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into GrafanaUpdates

Ja-Dunn committed:
Merge pull request #36257 from chetansriv/appgallerylisting

Update gallery listing doc

cherylmc committed:
update

Ja-Dunn committed:
Merge pull request #36614 from v-jerkin/master

Update author field in existing articles

v-geberr committed:
edits

PRMerger16 committed:
Merge pull request #36607 from MicrosoftGuyJFlo/AzureDocsIssue6460

[MFA] GitHub Issue 6460 Modern auth and Office 2013

bandersmsft committed:
azure policy - adding Policy Insights article

Commit:
Azure Redis-Cache TOC Conversion Request

curtand committed:
resolve merge conflict

Commit:
Azure CDN TOC Conversion Request

PRMerger17 committed:
Merge pull request #36601 from cherylmc/ub

ubiquiti

SnehaGunda committed:
Revertng "Updating token details" commit

This reverts commit 42d67e27f9e8962778c0fce35e99816dfec08a8e.

v-geberr committed:
update screen shots for new UX

gokhanuluderya-msft committed:
added know issues note for UI pointing to python environment set in the environment variable.

Ja-Dunn committed:
Merge pull request #36619 from cmcclister/1221279

Azure Redis-Cache TOC Conversion Request

PRMerger20 committed:
Merge pull request #36612 from v-geberr/0329-concept-cleanup

0329 concept cleanup

Ja-Dunn committed:
Merge pull request #36623 from v-geberr/0329-fix-images

update screen shots for new UX

Ja-Dunn committed:
Merge pull request #36621 from cmcclister/1221374

Azure CDN TOC Conversion Request

Commit:
Merge pull request #36622 from gokhanuluderya-msft/master

added know issues note for UI pointing to python environment set in t…

ktoliver committed:
Merge pull request #36618 from bandersmsft/master

azure policy - adding Policy Insights article

Ja-Dunn committed:
Merge pull request #36605 from SnehaGunda/CL

Removing editor

v-geberr committed:
remove fuzzy search, switch out new ddl

Ja-Dunn committed:
Merge pull request #36616 from anirudhcavale/GrafanaUpdates

Grafana updates

iainfoulds committed:
[VM] Updating for simplified PowerShell cmdlets

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vmpackernewposh

iainfoulds committed:
Merge branch 'vmpackernewposh' of https://github.com/iainfoulds/azure-docs-pr into vmpackernewposh

zhiweiwangmsft committed:
Add password hash sync

zhiweiwangmsft committed:
minor

gsacavdm committed:
Update ASE limitation

v-geberr committed:
remove fuzzy search, update ddl

Commit:
Task 1045: Add token auth and standard-cache info

PRMerger17 committed:
Merge pull request #36615 from curtand/grouplic0329

[Azure AD LICENSING] add preview boilerplate

PRMerger17 committed:
Merge pull request #36636 from gsacavdm/patch-2

Update ASE limitation

GitHubber17 committed:
Merge pull request #36630 from v-geberr/0329-screenshot-fix-03

remove fuzzy search, switch out new ddl

iainfoulds committed:
[VM] Updating broken link

GitHubber17 committed:
Merge pull request #36603 from iainfoulds/vmpackernewposh

[VM] Build Windows VM image with Packer

Commit:
Merge pull request #36634 from zhiweiwangmsft/patch-28

Add password hash sync

GitHubber17 committed:
Merge pull request #36638 from v-geberr/0329-screenshote-fix-04

remove fuzzy search, update ddl

GitHubber17 committed:
Merge pull request #36639 from dksimpson/dean-1045

Task 1045: Add token auth and standard-cache info

GitHubber17 committed:
Merge pull request #36596 from mimig1/03292018bi

Cosmos DB: Updated screenshots DocDB->Cosmos DB

GitHubber17 committed:
Merge pull request #36632 from iainfoulds/vmscalesettutorialnewposh

[VM] VM Windows scale set updates

PRMerger13 committed:
Merge pull request #36302 from JimacoMS2/sf-set-up-linux-env-fix2

added info about support and apt-transport-https and removed WSL from…

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

tynevi committed:
Merge pull request #36029 from mimig1/03222018-scrub2

Cosmos DB: Removed media folder from unused documentdb folder

v-alje committed:
Merge pull request #36648 from MicrosoftDocs/FromPublicRepo

From public repo
